app/views/custom/layouts/_footer.html.erb

```diff
@@ -0,0 +1,38 @@
+<footer>
+  <div class="row">
+    <div class="small-12 large-4 column">
+      <h1 class="logo">
+        <%= link_to "https://pro-ethics.eu", title: t("layouts.footer.framework") do %>
+          <%= image_tag "ProEthics.svg", width: "250px" %>
+        <% end %>
+      </h1>
+
+      <p class="info">
+        <%= sanitize(t("layouts.footer.description",
+          open_source: link_to(t("layouts.footer.open_source"), t("layouts.footer.open_source_url"), target: "blank", rel: "nofollow"),
+          consul:  link_to(t("layouts.footer.consul"), t("layouts.footer.consul_url"), target: "blank", rel: "nofollow"))) %>
+        <%= t("layouts.footer.contact_us") %>
+      </p>
+    </div>
+
+    <div class="footer-sections small-12 large-8 column">
+      <div class="small-12 medium-10 column">
+        <%= image_tag "EU-EC-Logo", style: "height: 65px !important" %>
```

Comment:
I think we need an `alt` attribute here; otherwise, it's invalid HTML and an accessibility issue.